### ISSUE TITLE
Edit Dashboard

### ISSUE THREAD

**Issue by jcourcha** [campai-bv/autotask-dashboards-os#61]
When I edit a dashboard to remove Missing Issue Type, it does not show all the widgets and I have to create a new dashboard.

**Comment by coencoppens** [comment]
Sounds like this might be related to issue #59 . Working on it, I'll keep you posted.

Btw, are you running 1.5.1. too?

**Comment by jcourcha** [comment]
Yes running 1.5.1.  No error just odd behavior with the editing. Thanks!
